### PR TITLE
(EJ2-834303): Added new samples for chart and stockchart.

### PR DESCRIPTION
**Pull request by Soundeswari-Selvam**
**Description:**

Added UG for the volume 2 release features and improvement in Chart and Stock Chart.

### CODE REVIEW

File: ej2-javascript/chart/chart-appearance.md [hunk 1]
```diff
@@ -331,6 +331,97 @@ Chart can be given a title using [`title`](../api/chart/#title-string) property,
 {% previewsample "page.domainurl/code-snippet/chart/chart-appearance-cs15" %}
 {% endif %}
 
+### Position and Alignment
+
+By using the `position` property, you can position the `title` at left, right, top or bottom of the chart. The title is positioned at the top of the chart, by default.
```
Comment (Gowrimathi) [inline ej2-javascript/chart/chart-appearance.md:336]: By using the `position` property in `titleStyle`, you can position the `title` at left

**Review by Gowrimathi — CHANGES_REQUESTED** [review]
Requested review changes

File: ej2-javascript/chart/Chart-types/pare-to.md [hunk 2]
```diff
@@ -43,6 +43,37 @@ The initial values are represented by column chart and the cumulative values are
 {% previewsample "page.domainurl/code-snippet/chart/chart-types-cs107" %}
 {% endif %}
 
+## Pareto customization
+
+The pareto line series can be customized by using the [`marker`](../api/chart/paretoOptions/#marker), [`width`](../api/chart/paretoOptions/#width), [`dashArray`](../api/chart/paretoOptions/#dasharray), and [`fill`](../api/chart/paretoOptions/#fill) properties in the paretoOptions. The secondary axis for the pareto series can be shown or hidden using the [`showAxis`](../api/chart/paretoOptions/#showaxis) property.
```
Comment (harivenkateshe) [inline ej2-javascript/chart/Chart-types/pare-to.md:48]: For the "paretoOptions", API reference link is missing.

File: ej2-javascript/stock-chart/tool-tip.md [hunk 3]
```diff
@@ -81,6 +81,37 @@ By default, tooltip shows information of x and y value in points. In addition to
 {% previewsample "page.domainurl/code-snippet/stock-chart/getting-started-cs29" %}
 {% endif %}
 
+## Position the Tooltip
+
+By using the [`position`](../api/stock-chart/stockTooltipSettings/#position) property, you can fix the tooltip in one position or have it move along with the mouse. By default, the tooltip is positioned at the left of the stock chart.
```
Comment (harivenkateshe) [inline ej2-javascript/stock-chart/tool-tip.md:86]: Check and correct this sentence - "By default, the tooltip is positioned at the left of the stock chart."

**Review by Gowrimathi — APPROVED** [review]
Fine

**Pull request merged**